docs/src/pages/docs/crocks/Equiv.md

```diff
@@ -58,6 +58,39 @@ eq.contramap(length)
 
 </article>
 
+<article id="topic-construction">
+
+## Construction
+
+```haskell
+Equiv :: ((a, a) -> Boolean)-> Equiv a a
+```
+
+The constructor for `Equiv` is a unary function that takes a binary predicate
+function as its argument. The binary function's arguments should be the same
+type, although a union of multiple types can be used.
```

Comment:
Well they *should* be, the reason we do not check specifically is because if could have a function that takes either a `Function` or an `Object`. So that really represents the type `Function + String` which is the union of those types, there is no way to capture adhoc unions like that, so I went with *should*

Reply:
@bennypowers 
I will add an example of this. One of the things about `Equiv` is it is not strictly the same thing, but compares things to see if they are equivalent. So maybe some `Number` is equivalent  to some `Object`. Say we have two functions that take in `Number + Object`, the result of calling one with the `Number` and the other with the `Object` results in an Equivalent result (this is called a pullback).

So Equiv is in there because it is different then the way we deal with predicates. Most of the predicates we use are one type, this allows us to move from strict equalities (or inequalities) to equivalence for our comparisons.

But I think I may need to show an example of this in these docs